topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -0,0 +1,290 @@
+>
+{title="Exploring Configuration Options"}
+
+### Setting up repositories
+
+{id="configuration.repositories"}
+
+All IntelliJ Platform SDK artifacts are available via IntelliJ Maven repositories (see [](intellij_artifacts.md)), which exist in three variants:
+
+- releases
+- snapshots
+- nightly (only select artifacts)
+
+All required repositories for the project must be declared explicitly within the `intellijPlatform` extension:
+
+```kotlin
+repositories {
+  mavenCentral()
+
+  intellijPlatform {
+    releases()
+    snapshots()
+    nightly()
+  }
+}
+```
+
+The `intellijPlatform` extension available for the `repositories {}` provides the following helper methods for adding repositories:
+
+| Name                          | Description                                                                                                                                       |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| `releases`                    | IntelliJ Platform Releases repository.                                                                                                            |
+| `snapshots`                   | IntelliJ Platform Snapshots repository.                                                                                                           |
+| `nightly`                     | IntelliJ Platform Nightly repository, not available publicly.                                                                                     |
+| `ivy`                         | Local Ivy repository for resolving local Ivy XML files used for describing artifacts like local IntelliJ Platform instance, bundled plugins, etc. |
+| `marketplace`                 | JetBrains Marketplace Repository, hosts plugins for IntelliJ-based IDEs.                                                                          |
+| `jetbrainsRuntime`            | JetBrains Runtime (JBR), CloudFront-based hosting from which specific JBR releases are fetched, if explicitly requested.                          |
+| `binaryReleasesAndroidStudio` | Android Studio Binary Releases, required when running IntelliJ Plugin Verifier against Android Studio releases.                                   |
+| `binaryReleases`              | IntelliJ IDEA Binary Releases, required when running IntelliJ Plugin Verifier against JetBrains IntelliJ-based IDE releases.                      |
+| `recommended`                 | Applies a set of recommended repositories.                                                                                                        |
+
+#### Dependency Resolution Management
+
+{id="configuration.dependencyResolutionManagement"}
+
+To access the IntelliJ Platform Gradle Plugin within the <path>settings.gradle.kts</path> to use with `dependencyResolutionManagement`, add:
```

Comment:
"To access ... to use..." sounds weird. Also, I'm not sure I understand this section completely. Why exactly do I need this?